fem/restriction.cpp

```diff
@@ -1520,7 +1665,7 @@ void L2FaceRestriction::SetFaceDofsScatterIndices1(
    const int dim = fes.GetMesh()->Dimension();
    const int dof1d = fes.GetFE(0)->GetOrder()+1;
    const int elem_index = face.element[0].index;
-   GetFaceDofs(dim, face_id1, dof1d, face_map); // Only for quad and hex
+   GetFaceDofs_H1_L2(dim, face_id1, dof1d, face_map); // Only for quad and hex
```

Comment:
Is it possible for this code path to be reached for a face that is not a quad? Can the unsupported cases be prevented or checked?

Reply:
It should not be possible, the checks in `H1_ND_RT_FaceRestriction::CheckFESpace` should fail in the unsupported cases